src/seedu/addressbook/AddressBook.java

```diff
@@ -209,7 +191,7 @@
     public static void main(String[] args) {
         showWelcomeMessage();
         processProgramArgs(args);
-        loadDataFromStorage();
+        initialiseAddressBookModel(loadPersonsFromFile(storageFilePath));
```

Comment:
maybe can separate this into 2 lines and use a variable to explain what you get after loading persons from file. :)